packages/frontend-2/pages/authn/sso/index.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <AuthSsoLogin />
+</template>
+
+<script setup lang="ts">
+useHead({
+  title: 'Log in'
```

Comment:
Maybe check with Michal or Benjamin, but we are mixing `Log in` and `Sign in`, maybe good to be consistent :)